hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/iamwhoamianamy/otus-golang-homework/tree/master/hw01_hello_otus
```

```suggestion
module github.com/iamwhoamianamy/otus-golang-homework/hw01_hello_otus
```
`tree/master/` - это часть роутинга UI в Github, т.е. не имеет отношения к Go-модулям